code/modules/research/designs/autolathe_desings/autolathe_designs_sec_and_hacked.dm

```diff
@@ -73,46 +84,88 @@
 	build_path = /obj/item/restraints/handcuffs
 	category = list("hacked", "Security")
 
+/*
 /datum/design/receiver
 	name = "Modular Receiver"
 	id = "receiver"
 	build_type = AUTOLATHE | NO_PUBLIC_LATHE
 	materials = list(/datum/material/iron = 15000)
 	build_path = /obj/item/weaponcrafting/receiver
 	category = list("hacked", "Security")
+Old receiver, removed, you now use different receivers found in guncrafting.dm
+*/
+/datum/design/rifle_receiver
+	name = "Rifle Receiver"
+	id = "rifle_receiver"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron = 40000)
+	build_path = /obj/item/weaponcrafting/improvised_parts/rifle_receiver
+	category = list("hacked", "Security")
 
 /datum/design/shotgun_slug
 	name = "Shotgun Slug"
-	id = "shotgun_slug"
+	id = "12g shotgun_slug"
```

Comment:
I'll be honest, I'd keep these to seclathe now. If the bartender wants lethal 12g he should be asking security.

Reply:
These were already in the hacked autolathe.
Like a derp I changed the ID instead of the name though, I meant to change the name to 12g so it'd be a bit more consistent in the menu.
Removing 12g lethals from the hacked autolathe is something Kevinz would have to weigh in on.